example/_config.yml

```diff
@@ -8,6 +8,9 @@ description: > # this means to ignore newlines until "baseurl:"
 baseurl: "/minima"
 twitter_username: jekyllrb
 github_username:  jekyll
+# if you want to use disqus as comment system, be aware that it only works on production
+disqus:
+  shortname: your_disqus_shortname
```

Comment:
Couldn't we limit the output to production via liquid, and make it opt-in entirely by commeting out the config (and making it dependent on the config existing)?